src/app/in-memory-data.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { InMemoryDbService } from 'angular-in-memory-web-api';
+import { Hero } from './hero';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class InMemoryDataService implements InMemoryDbService {
+  createDb() {
+    const heroes = [
+      { id: 11, name: 'Dr Zef' },
+      { id: 12, name: 'Cpt Mountain Dew' },
+      { id: 13, name: 'Dr DocX' },
+      { id: 14, name: 'MassiveClint' },
+      { id: 15, name: 'Mr Angular' },
+      { id: 16, name: 'Pickle Rick' },
+      { id: 17, name: 'Jetman JPEG' },
+      { id: 18, name: 'Backpack man' },
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/infinity-arc/Kobus-tour-of-heroes/pullRequest?prid=7838883)
```suggestion
      { id: 18, name: "Backpack man" },
```